Model/RatepayInvoicePaymentMethod.php

```diff
@@ -0,0 +1,131 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+
+namespace Wirecard\Oxid\Model;
+
+use Wirecard\Oxid\Core\Helper;
+use Wirecard\PaymentSdk\Config\PaymentMethodConfig;
+use Wirecard\PaymentSdk\Transaction\RatepayInvoiceTransaction;
+
+/**
+ * Payment method implementation for Ratepay Invoice
+ *
+ * @since 1.2.0
+ */
+class RatepayInvoicePaymentMethod extends PaymentMethod
+{
+    /**
+     * @inheritdoc
+     *
+     * @since 1.2.0
+     */
+    protected static $_sName = "ratepay-invoice";
+
+    /**
+     * @inheritdoc
+     *
+     * @return Config
+     *
+     * @since 1.2.0
+     */
+    public function getConfig()
+    {
+        $oConfig = parent::getConfig();
+
+        $oPaymentMethodConfig = new PaymentMethodConfig(
+            RatepayInvoiceTransaction::NAME,
+            $this->_oPayment->oxpayments__wdoxidee_maid->value,
+            $this->_oPayment->oxpayments__wdoxidee_secret->value
+        );
+
+        $oConfig->add($oPaymentMethodConfig);
+        return $oConfig;
+    }
+
+    /**
+     * @inheritdoc
+     *
+     * @return RatepayInvoiceTransaction
+     *
+     * @since 1.2.0
+     */
+    public function getTransaction()
+    {
+        return new RatepayInvoiceTransaction();
+    }
+
+    /**
+     * @inheritdoc
+     *
+     * @return array
+     *
+     * @since 1.2.0
+     */
+    public function getConfigFields()
```

Comment:
Im missing, config fields for configuring the allowed currencies, shipping, billing countries, if billing_shipping needs to be the same, min/max basket amount

Reply:
As mentioned in the PR description we only added the basic configuration for this payment method and intentionally left the currencies and shipping, billing address settings out for now to have a common ground to get started with the development.

Min/max basket amount can be handled automatically through OXID's payment method configuration.